### DISCUSSION TITLE
Mark content as "Read"

### DISCUSSION THREAD

**Discussion by GuyDev1** [dailydotdev/daily#836] [Feature Requests]
I know there's already an option to "hide" the content, though I assume the algorithm treats it like content I don't want to read. Whereas a "read" indication simply clears content that I've already read. so I can be exposed to new things.

makes sense?

**Comment by idoshamun** [comment]
Hi @GuyDev1,

Right now, the algorithm doesn't penalize "hide" events. We're working on downvoting, which should bring the penalization aspect.
But on the other hand, marked as read can indicate this article needs to be rewarded.
I'll discuss it internally, thanks a lot!

**Reply by GuyDev1** [reply]
Sounds good, thanks!

**Reply by nimrodkra** [reply]
@GuyDev1 downvoting is now live! 🎉 [Check it out](https://app.daily.dev/posts/QK301jNUY)

**Comment by baggiponte** [comment]
Hi, I started using daily.dev yesterday. I like the tool - a bit like a HN on steroids. I have been reading tech news for a couple of years now and I would like to see a "already read" button to hide the content but still reward the algorithm.

**Reply by idoshamun** [reply]
@baggiponte if you hit the upvote button it will reward the algorithm and hide this post from your feed after several minutes

**Comment by fResult** [comment]
Is there any "Mark as Read" feature right now?
For me, I prefer "Mark as Read" rather than just "Hide", because the blog post that I have to "Hide", I may still need to read it again.

Sometimes I may not want to add upvote/downvote to the blog post because I feel neutral for that time.
So, I still need the "Mark as Read" feature.

Thank you. 🙏

**Comment by banezia** [comment]
Hi! I'm currently enjoying a lot the app. Thanks for creating it!

I wanted to ask for a feature somewhat like the one described here. 

I was thinking that it could be great if there were some sort of filter or toggler where one as an user can activate it and then stop seeing posts already "viewed”. These "viewed posts" constitute the posts present in the scrollable part (the feed) before triggering the infinite scroller, whether they were clicked or not, so here the feature would be to hide these posts. 


If it is still not clear, I will try to demonstrate it with a completely flawed and basic implementation I was thinking:


In a given moment, when the user start consuming the feed, the first batch of posts loaded, before loading the next batch mark all the previous as read, even if not clicked. Here the marked posts can be stored by post id (?) in a cookie.

So with that the next time the user, for example, reload the web page, when loading the feed posts, the marked posts are "ignored" and not shown.

Thank you in advance!

**Reply by idoshamun** [reply]
Hi @Estebanrg21,

We already do that and hide viewed posts. The only difference is that we do it after 2 times, not the first

**Reply by banezia** [reply]
Awesome, thank you!

**Comment by preslavPA** [comment]
Hello. I'd like to share the same sentiment as @fResult. It'd be great to have a "Mark as read" feature which doesn't hide the posts or even an automatic indication which is visible once the post is read. For me personally that would be very useful for the bookmarked posts as what I tend to do is bookmark several posts at a time and then read them later. That way I could keep track of which posts I still need to read.

Thanks in advance. Love daily.dev keep up the good work!

**Reply by idoshamun** [reply]
Hi @preslavPA,

Have you tried our bookmarks feature? Because I'm not sure what you refer to exactly in terms of flow and how the "mark as read" will help here

**Reply by preslavPA** [reply]
Hi @idoshamun,

I just realized that on the desktop version the read articles have a slightly darker background which is what I need actually. But, I cannot see the same behaviour on mobile. Could that be applied to the mobile view as well?

To answer your question, Yes, I'm using the bookmarks feature daily. What I'm refering to is not specifically tied to the bookmarks feature, I was just giving an example.

**Reply by idoshamun** [reply]
I see. Yes our mobile view doesn't have a different UI for read posts
